Components/RectangleButton.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import { TouchableRipple } from 'react-native-paper';
+
+export default class RectangleButton extends React.Component {
+    render() {
+        return(
+            <TouchableRipple
+              onPress={() => this.props.buttonFunction()}
```

Comment:
`onPress={this.props.buttonFunction}` instead. Also, call it `this.props.onPress`